test/Select-test.js

```diff
@@ -2129,6 +2129,66 @@ describe('Select', () => {
 
 	});
 
+	describe('with multi=true and onSelectResetsInput=false', () => {
```

Comment:
more accurately `describe('with multi=true and different onSelectResetsInput', () => {`